anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
I've made a deck with 1 note / 2 cards. You can download it here: http://www.speedyshare.com/vFvKQ/tmp.apkg
{{c1::Ens estan construint un edifici de vuit pisos[sound:catalan_020_S06.mp3]}} {{c2::i  de vista ja no en queda[sound:catalan_020_S07.mp3]}}!
 It doesn't do what you say. In question 1, I hear the recording of the second part of the sentence, in question 2, I hear the first part of the sentence.
In both answers, I hear the entire sentence. Exactly what I expect.